sdk/analytics/src/main/java/com/klaviyo/analytics/networking/KlaviyoApiClient.kt

```diff
@@ -318,10 +332,22 @@ internal object KlaviyoApiClient : ApiClient {
          * Re-queue the job to run again after [flushInterval] milliseconds
          */
         private fun requeue() {
-            Registry.log.verbose("Retrying network batch in $flushInterval")
+            Registry.log.verbose("Network batch will run in $flushInterval ms")
             force = false
+            enqueuedTime = Registry.clock.currentTimeMillis()
             handler?.postDelayed(this, flushInterval)
         }
+
+        private fun computeRetryInterval(attempts: Int): Long {
```

Comment:
Logic here looks totally sound. Just wondering if this is a mirror of what we do on iOS? Like are the limits we apply to our jitter the same, etc.

Reply:
Default min/max retry intervals as well

Reply:
Yes I believe so. Would like a confirmation of my reading from Noah or Ajay. The difference is mostly just that I compute the retry time all in one function, whereas on iOS its split up into a couple different parts of the network queueing code.

Reply:
@ndurell @ajaysubra when one of you has a chance ^

Reply:
Thanks for the verification!